static-site/src/components/layout.jsx

```diff
@@ -23,6 +23,9 @@ export default class MainLayout extends React.Component {
           <title>{`${siteTitle}`}</title>
           <meta name="description" content={siteDescription} />
           <link rel="me" href="https://mstdn.science/@nextstrain" />
+          <link href={`${siteUrl}/blog/atom.xml`} rel="alternate" title="Atom feed for nextstrain.org/blog" type="application/atom+xml" />
+          <link href={`${siteUrl}/blog/feed.json`} rel="alternate" title="JSON feed for nextstrain.org/blog" type="application/json" />
+          <link href={`${siteUrl}/blog/rss2.xml`} rel="alternate" title="RSS2 feed for nextstrain.org/blog" type="application/rss+xml" />
```

Comment:
_non-blocking_

All these (including mastodon link) should be conditioned on SiteConfig's `groupsApp` being `false`, but no need to block this PR on that.

Reply:
done.